mindpype/kernels/baseline_correction.py

```diff
@@ -12,6 +12,12 @@ class BaselineCorrectionKernel(Kernel):
     the mean of the baseline period from the input data
     along a specified axis.
 
+    .. note::
+        This kernel utilizes the numpy functions
+        :func:`asarray <numpy:numpy.asarray>`,
```

Comment:
We can stick to referring only to the primary functions (in this case, mean) and remove the lines referring to utilities like asarray and array